runtime/zephir.c

```diff
@@ -161,10 +161,9 @@ static void zephir_compile_program(zval *program TSRMLS_DC)
  */
 static void zephir_parse_file(const char *file_name TSRMLS_DC)
 {
-    char *file_name_pass = (char*) file_name;
-	char *contents;
+	char *file_name_pass = (char*) file_name;
+	zend_string *contents;
```

Comment:
[Why I change]

Because PHP7 introduce `zend_string` and `php_stream_copy_to_mem` return `zend_string`